Example/SnapshotTests/BPKRatingSnapshotTest.m

```diff
@@ -33,7 +33,7 @@ - (void)setUp {
 
 - (void)testRatingOutOfRangeLow {
     BPKRating *bpkRating = [[BPKRating alloc] initWithFrame:CGRectZero];
-    bpkRating.title = @"Capped";
+    bpkRating.title = [[BPKRatingTextDefinition alloc] initWithHighRatingText:@"" mediumRatingText:@"" lowRatingText:@"Capped"];
```

Comment:
similar to my comment above, could we have a couple of pre-defined definitions that we use for most of these?